### DISCUSSION TITLE
MAUI Roadmap

### DISCUSSION THREAD

**Discussion by hot33331** [getsentry/sentry-dotnet#2199] [Q&A] [answered]
Where can I find the Roadmap for Sentry in regards to .NET MAUI? My team is asking which features are planned and it would be cool to have a link where I can see what is on the road ...

**Comment by mattjohnsonpint** [comment, marked as answer]
Thanks for bringing this up.  I've cleaned up and repurposed our original MAUI meta-issue and updated it with the current plan.  Please see: #1174

If you have any specific feature requests, we'd love to hear them.  Please comment over there.  Thanks!

**Reply by hot33331** [reply]
Thank you! From the top of my head automatic performance measurement hot and cold start of the app, navigation ...
